docs/ReleaseHistory.md

```diff
@@ -12,6 +12,10 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- BRK: `SecretLiteral` and `RegexPattern.GetDetections` are no longer public.
```

Comment:
OK, so people can't instantiate and search using a single pattern instance anymore. This allows us to obscure the new string input magic that we wouldn't want people to have to struggle to construct anyway (or at least it avoids the sensible question of why there's no string supporting helper). As a result, users now are required to initialize a full masker instance with their pattern(s) or interest to get results. That all seems fine and there's a sensible workaround if we, in fact, break people by dropping this visibility.